next/content/docs/guides/vercel.md

```diff
@@ -1,73 +1,133 @@
+
+Neon’s branching feature addresses all of these challenges. With Neon, a branch is a copy-on-write clone of your data that can be created instantly, which makes Neon's database branches a scalable and cost-effective solution for preview deployments. After adding the integration, each preview deployment will automatically have its own isolated, production-like database.
+
+To use the integration with preview deployments:
+
+1. Add the Neon integration to your Vercel project, as described above.
+2. Ensure that the database connection settings in your application correspond to the environment variable settings configured by the Neon integration. For example, if your applications's database connection is defined by a `DATABASE_URL` variable, make sure that setting in your application corresponds to the `DATABASE_URL` setting configured by the Neon integration. You can find the environment variable settings in Vercel by navigating to the Vercel dashboard, selecting your project, and selecting **Settings** > **Environment Variables**. In addition to `DATABASE_URL` variable, the integration sets the following environment variables, which may be used by some application frameworks:
+
+    - `PGHOST`
+    - `PGUSER`
+    - `PGDATABASE`
+    - `PGPASSWORD`
+
+    <Admonition type="note">
+    After adding the integration and setting the environment variables in your application, you may have to redeploy your application in Vercel for the environment variables set by the integration to take effect.
+    </Admonition>
+
+3. After the integration is added and the environment variables in your application are configured to match those in your Vercel project, each Git branch pushed to the GitHub account associated with your Vercel project creates a branch in Neon and automatically configures the environment variables in your Vercel preview environment, allowing the preview deployment to connect to the database the branch.
+
+### Managing preview deployment branches
 
-## Need help?
+Branches created in Neon for preview deployments are not removed automatically, so they may start accumulating in your Neon project after a period of time. You can manage the removal of old branches in the Neon Console or you build this task into to your CI/CD pipeline using the [Neon API](https://neon.tech/api-reference/v2). See [Manage branches](/docs/manage/branches) for information about how to remove branches using the console or API.
 
-Send a request to [support@neon.tech](mailto:support@neon.tech), or join the [Neon community forum](https://community.neon.tech/).
+It is important to remove old and unused branches. Branches hold a lock on the data they contain, preventing disk space from being reallocated, which can lead to excessive disk space consumption. The Neon Free Tier limits the point-in-time restore window for a project to 7 days. To minimize disk space usage and stay within the Free Tier limits, avoid allowing branches to age beyond the 7 day limit.
```

Comment:
not so sure about this paragraph... I mean it's sort of our fault, lack of good ux, that we don't manage old branches

Reply:
OK. We'll remove it for now. We can look at it addressing it in a next iteration.